<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLectureShedule</class>
 <widget class="QDialog" name="AddLectureShedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sitka Banner</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AddLectureShedule</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/ghjk.png</normaloff>:/images/ghjk.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Banner</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Add Lecture Shedule</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>511</width>
     <height>340</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="verticalSpacing">
     <number>13</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTimeEdit" name="StartTime">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>To</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTimeEdit" name="EndTime">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Monday</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="Monday">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Tuesday</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="Tuesday">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Wednesday</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="Wednesday">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Thursday</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="Thursday">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Friday</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="Friday">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="LecTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>721</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>460</y>
     <width>317</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="SaveLec">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(122, 1, 142, 255));
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ResetLec">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(122, 1, 142, 255));
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>-100</x>
     <y>-1</y>
     <width>1251</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/6070859.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BackLec">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>83</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/left-arrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>label</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>LecTable</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>BackLec</zorder>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
